client/src/components/itemColorOptions.jsx

```diff
@@ -8,20 +7,13 @@ const ItemColorOptions = (props) => {
 
   const [color, setColor] = useState(props.color);
 
-  //handle box clicks which will take in productId and return productId
-
-  //pass productId up to app.jsx
   return (
     <div>
       <SectionDescriptionText>Color:</SectionDescriptionText> <b>{color}</b>
       <br></br>
       <ColorBoxBlack onClick={() => props.handleColorBoxClick(props.similarItems[0])} onMouseEnter={() => setColor('Black')} onMouseLeave={() => setColor(props.color)}></ColorBoxBlack>
```

Comment:
Are these for color of product ?

Reply:
Yeah I only have the same three color options for each item. So these colors correspond to a product.